src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
 // write code here
+
+const listaUl = document.querySelector('ul');
+const listaLi = listaUl.querySelectorAll('li');
+
+const posortowanaLista = [...listaLi].sort((a, b) => {
+  const salaryA = Number(a.getAttribute('data-salary').replace(/[$,]/g, ''));
+  const salaryB = Number(b.getAttribute('data-salary').replace(/[$,]/g, ''));
```

Comment:
Similarly to the previous comment, ensure that 'b.getAttribute('data-salary')' is not null before using the 'replace' method.